django/core/files/utils.py

```diff
@@ -5,20 +5,21 @@
 
 
 def validate_file_name(name, allow_relative_path=False):
+    path = str(name).replace("\\", "/")
```

Comment:
Is this line part of just `Added extra tests for django.core.files.utils.validate_file_name helper.`?

Reply:
Yes, indeed. When I added the extra tests, assertions for these failed:

* For filenames:
```
\\tmp\\..
\\tmp\\.
```
* For dirnames, both when `allow_relative_path` is True and False:
```
tmp\\..\\path
..\\path
\\tmp\\..\\path
\\tmp\\path
```

So I added the `replace` call since it is already present in the call sites. Do you think it would be better to not add it in this commit (and remove the failing test paths), and then have a new "2.5" commit where the replace and extra test cases are added?

Reply:
I'm notoriously slack about these things, so I'd probably merge these three as a single cleanup 😅 (but perhaps get a second opinion). 

They're all one related change to me. Refactor and increase test coverage for...

But the three commits here make the review easy.